cdroot.overlay/isolinux/menu.cfg

```diff
@@ -2,13 +2,13 @@ default cold-offline
 label cold-offline
   menu label cold-offline: create wallet, sign transactions
   kernel /casper/vmlinuz
-  append boot=casper initrd=/casper/initrd.gz root=/dev/ram rw showmounts debug toram bitkey=cold-offline --
+  append boot=casper initrd=/casper/initrd.gz root=/dev/ram rw showmounts toram bitkey=cold-offline --
```

Comment:
When debug is specified, systemd is flooding the console with messages during the boot process.